perfkitbenchmarker/providers/gcp/gcp_tpu.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2017 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module containing class for GCP's cloud TPU.
+
+cloud TPU can be created and deleted.
+"""
+
+import json
+import logging
+
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import resource
+from perfkitbenchmarker.providers.gcp import util
+
+DEFAULT_CLOUD_SPANNER_CONFIG = 'regional-us-central1'
+TPU_TIMEOUT = 1200
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string('tpu_version', 'nightly',
+                    'The version for the Cloud TPU.')
+
+
+class GcpCloudTpu(resource.BaseResource):
+  """class representing a GCP cloud TPU.
+
+  Attributes:
+    name: Name of the cloud TPU to create.
+    project: the GCP project.
```

Comment:
Would you ever create a TPU in a different project than what the rest of PKB is using?

Reply:
The GCE and TPU use the same same project.

Reply:
Ok